### PR TITLE
Tweak light mode background colors.

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Adjust light mode background color heirarchy to be more consisitent with dark mode styles

**Description**

### CODE REVIEW

**Comment by defunctzombie** [comment]
> Adjust light mode background color heirarchy to be more consisitent

Consistent with what?

**Comment by 2metres** [comment]
> Consistent with what?

Dark mode

**Comment by jtbandes** [comment]
Looks like we are losing the grid on the legacy plot panel in light mode:

image

**Comment by jtbandes** [comment]
You've got some test failures:

> Warning: React does not recognize the `shouldShow` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `shouldshow` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
[61](https://github.com/foxglove/studio/runs/6281133224?check_suite_focus=true#step:6:61)

**Comment by 2metres** [comment]
@amacneil will address the tabs in a separate PR

**Review by jtbandes — APPROVED** [review]

File: packages/studio-base/src/components/PanelToolbar/index.stories.tsx [hunk 1]
```diff
@@ -41,9 +42,15 @@ class MosaicWrapper extends React.Component<{
             renderPreview={() => undefined as any}
           >
             <HelpInfoProvider>
-              <div style={{ width, height: 300, padding: 30, position: "relative" }}>
-                {id === "Sibling" ? "Sibling Panel" : this.props.children}
-              </div>
+              <Box
+                width="100%"
```
Comment (defunctzombie) [inline packages/studio-base/src/components/PanelToolbar/index.stories.tsx:46]: Didn't we determine box is slow? we should be using divs with useStyle

File: packages/studio-base/src/components/PanelToolbar/index.stories.tsx [hunk 2]
```diff
@@ -135,7 +142,7 @@ storiesOf("components/PanelToolbar", module)
       </Icon>
     );
     return (
-      <MosaicWrapper width={500}>
+      <MosaicWrapper width={468}>
```
Comment (defunctzombie) [inline packages/studio-base/src/components/PanelToolbar/index.stories.tsx:145]: What do these have to do with light mode?

File: packages/studio-base/src/components/PanelToolbar/index.stories.tsx (hunk 1 above)
Comment (2metres) [reply packages/studio-base/src/components/PanelToolbar/index.stories.tsx:46]: For stories I feel like Box is fine.

File: packages/studio-base/src/components/PanelToolbar/index.stories.tsx (hunk 2 above)
Comment (2metres) [reply packages/studio-base/src/components/PanelToolbar/index.stories.tsx:145]: The spacing on the PanelToolbar changed this is attempting to keep the visual diff as similar as possible in the stoybook

**Pull request merged**